lib/main.dart

```diff
@@ -20,37 +27,63 @@ class SeedsApp extends StatefulWidget {
   _SeedsAppState createState() => _SeedsAppState();
 }
 
-class _SeedsAppState extends State<SeedsApp> {
-  final AuthService authService = AuthService();
 
+class _SeedsAppState extends State<SeedsApp> {
   @override
   Widget build(BuildContext context) {
-    return ToolboxApp(
-      noItemsFoundWidget: Padding(
-        padding: const EdgeInsets.all(32),
-        child: SvgPicture.asset(R.noItemFound),
-      ),
-      child: MaterialApp(
-        debugShowCheckedModeBanner: false,
-        theme: ThemeData(
-          primarySwatch: primary,
-          tabBarTheme: TabBarTheme(
-            indicator: TabRoundedLineIndicator(
-              context,
-              indicatorColor: primary,
-            ),
-          ),
+    return MultiProvider(
+      providers: [
+        Provider.value(value: HttpService()),
+        ChangeNotifierProvider.value(value: AuthModel()),
```

Comment:
```suggestion
        ChangeNotifierProvider(create: (_) =>  AuthModel()),
```
see :https://pub.dev/packages/provider#usage

Reply:
**Correct**

```
AuthModel authNotifier = AuthModel();
ChangeNotifierProvider.value(
  value: authNotifier,
// provide existing object here without instantiation - otherwise it may have undesired behavior // because build function should be designed to call any moment without side effects
  child: ...
)
```

In my understanding code below is equivalent to code above:

```
ChangeNotifierProvider(
  create: (_) => AuthModel()
)
```

Reply:
yes, correct but in your code, you were creating a new instance every time.

Reply:
but using `create` is one line less and is lazily created, unlike the global variable.